api/MediaKeySystemConfiguration.json

```diff
@@ -63,10 +63,10 @@
               "version_added": "≤79"
             },
             "firefox": {
-              "version_added": null
+              "version_added": "50"
```

Comment:
I guess this is based on https://bugzilla.mozilla.org/show_bug.cgi?id=1278198, but that doesn't seem to be the relevant bug/change.

https://github.com/mozilla/gecko-dev/commit/51df37a7c8d42fa9e764d666a892df45bdabb761 seems to be the commit that introduced the plural `audioCapabilities`.

Can you check all "50" versions again?